### PR TITLE
styleguide fix issue with static content

### LINKED ISSUE #913: Styleguide not visually representative
Issues with the fonts particularly. The elife logo doesn't load either. Have a chat with @chugginselifesciences if you are picking this up.

### PR DESCRIPTION
**Pull request by jyenes**
#### Background
static content not being displayed properly on Styleguide, so renamed  to assets

What does this PR do?
closes #913

### CODE REVIEW

**Comment by jyenes** [comment]
fixed right now!

**Comment by jyenes** [comment]
at what point this `https://s3.amazonaws.com/ci-elife-xpub-styleguide/913_assets_styleguide/index.html#peoplepicker` is being built?
it seems is not picking the new build changes.

**Comment by jyenes** [comment]
@giorgiosironi directory structure is now correct, the issue we have right now is how the content is being served, is there anyway in S3 to resolve /assets to include the bucket name?

screenshot 2018-11-08 at 10 41 57

**Comment by giorgiosironi** [comment]
From the call:

- can't resolve `/assets` to a branch-specific folder while using a single bucket for all branches
- trying to change the path within the Webpack build
- reserve option is to generate a container (derived from `nginx`) with strange routing rules, but it's much more complex to maintain

File: styleguide.config.js [hunk 1]
```diff
@@ -1,4 +1,17 @@
 const path = require('path')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
```
Comment (diversemix) [inline styleguide.config.js]: Is there any reason this file lives in the root and not in `webpack` alongside the other config files?

Comment (diversemix) [reply styleguide.config.js]: Just to be clear the file `styleguide.config.js` ... expected it to be something like `webpack/styleguide.config.js`

Comment (jsms90) [reply styleguide.config.js]: Does your comment relate to this line of the file?

Or about the location of this whole file?

File: styleguide.config.js [hunk 2]
```diff
@@ -44,13 +57,14 @@ module.exports = {
   ],
   webpackConfig: {
```
Comment (jsms90) [inline styleguide.config.js]: @diversemix Is your comment suggesting that we move this piece of the styleguidist config out of this file, and put it under the `/webpack` directory?

File: styleguide.config.js (hunk 1 above)
Comment (diversemix) [reply styleguide.config.js]: The whole file

File: styleguide.config.js (hunk 2 above)
Comment (diversemix) [reply styleguide.config.js]: My question is, if it is webpack config - why doesn't it live in the webpack folder?

File: styleguide.config.js (hunk 1 above)
Comment (jsms90) [reply styleguide.config.js]: I think this whole file needs to stay in the root. But we are right now moving just the webpack portion of this whole styleguidist config file into the `/webpack` directory.

Does that sound better?

File: styleguide.config.js (hunk 2 above)
Comment (jsms90) [reply styleguide.config.js]: The webpack portion of this config file does now live in the webpack folder. But the rest of the styleguidist config is still in the root.

Let us know what you think after this latest commit :+1:

File: styleguide.config.js [hunk 3]
```diff
@@ -1,4 +1,17 @@
 const path = require('path')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const rules = require('./webpack/rules.js')
```
Comment (diversemix) [inline styleguide.config.js]: Lines 3-14 could live in `webpack/styleguide.rules.js` which in turn could be required on line 47 - this ensures all the webpack config lives in the webpack folder. Let me know if I am missing something?

**Pull request merged**